docs/source/architecture/poet.rst

```diff
@@ -21,6 +21,59 @@ use of a trusted execution environment, identity verification and blacklisting
 based on asymmetric key cryptography, and an additional set of election
 policies.
 
+.. START OF POET CONTENT MOVED FROM INTRODUCTION
```

Comment:
Remove

Reply:
@jsmitchell - Double-checking: I should remove the two comments ("... START OF POET CONTENT" and "... END OF POET CONTENT") but not the actual PoET content, right? 

If so, will do.  And sorry - these were WIP comments to myself as I was moving text around, but I forgot to delete those comments.

Reply:
yep, the .. comments is what I was referring to